tools/cli/commands/connect.py

```diff
@@ -52,6 +52,15 @@
     $ {0} {1} example-instance --zone us-central1-a""")
 
 
+# The list of web browsers that we don't want to automatically open.
+#
+# This is a subset of the canonical list of python browser types
+# defined here: https://docs.python.org/2/library/webbrowser.html
+unsupported_browsers = [
```

Comment:
There are two reasons I didn't do that:

1. That list is a lot longer
2. According to the Python docs, the combined list is hard-coded, so we don't have much risk of new ones showing up that we don't support.